conda_build/build.py

```diff
@@ -1026,6 +1026,16 @@ def tests_failed(m, move_broken, broken_dir, config):
 
 def check_external():
     if sys.platform.startswith('linux'):
+        ldd = external.find_executable('ldd')
+        if ldd is None:
+            sys.exit("""\
+Error:
+    Did not find 'ldd' in: %s
+    'ldd' is necessary for building conda packages on Linux with
+    relocatable ELF libraries.  Please install with your system package
```

Comment:
In other words, I suppose we could build Glibc if we wanted to, but do we really want to go down that road? Even if we did, how do we ensure we install the right version `ldd` for that Linux system? It is [possible]( http://www.linuxfromscratch.org/lfs/view/6.4/chapter06/glibc.html ) though since you were curious. Just unconvinced that using `ldd` from anything but the system is a good idea.